lib/files.c

```diff
@@ -729,6 +729,21 @@ bool files_load_bytes_from_buffer_or_file_or_stdin(char *input_buffer,
     return res;
 }
 
+bool files_save_ESYS_TR(ESYS_CONTEXT *ectx, ESYS_TR handle, const char *path) {
```

Comment:
this commit looks screwy, did something go wrong with re-writing history? Looks like this was just meant to update createak.